util/opentelemetry-util-tracer/tests/server_span_tracer_test.py

```diff
@@ -0,0 +1,43 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+from opentelemetry.util.tracer import ServerSpanTracer
+from opentelemetry.trace import SpanKind
+from opentelemetry import trace, context
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleSpanProcessor,
+)
+
+class ServerSpanTracerTest(TestCase):
+
+    def test_span_found_current_context(self):
```

Comment:
Also test for SERVER span creation when there is no span in current context.

Reply:
I agree. I actually edited my start_span function because I realized that `INVALID_SPAN` would be returned instead of `None` for `trace.get_current_span()`, so now I was able to test for SERVER span creation. However, I am now unsure how to test for INTERNAL span creation, since `trace.get_current_span()`keeps on returning an `INVALID_SPAN ` for everything I have tried. I am going to ask upstream how to fix this problem so that I can test for INTERNAL span creation